src/entryPoints/AuthModule/UserInfoScreen.res

```diff
@@ -23,6 +23,8 @@ let make = () => {
       let info = AuthUtils.getAuthInfo(dict->JSON.Encode.object)
       setAuthStatus(LoggedIn(Auth(info)))
       setIsSidebarDetails("isPinned", false->JSON.Encode.bool)
+      LocalStorage.removeItem("PRE_LOGIN_INFO")
+      LocalStorage.removeItem("email_token")
```

Comment:
write a common function to clear the data from localstorage.you can write in LogicUtils

Reply:
Added the function in the common file for local storage utils